examples/time_series_forecasting.ipynb

```diff
@@ -0,0 +1,472 @@
+{
```

Comment:
<p>Shouldn't <code>X_t</code>  be <code>X_sw</code>  ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/giotto-ai/giotto-tda/pull/475/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Fixed.